### DISCUSSION TITLE
Performance of the GIF-format.

### DISCUSSION THREAD

**Discussion by Farcrada** [SevenTV/Extension#56] [Ideas]
To my understanding there are two things concerning the GIF format:
- BTTV and 7TV both use the GIF-format for animated emotes
- GIF is a terrible format

Seen as 7TV is still in it's early stages it has merit to address this before the chance passes (if it hasn't already). The overheating a laptop meme is funny 'n all, but if we could manage to get the browser to not consume unnecessary resources and/or modernize animated emotes it would only improve the experience. 

On [FrankerFaceZ's github](https://github.com/FrankerFaceZ/FrankerFaceZ/issues/896) I had made (invalid as it may be) an issue about exactly this. Where I absentmindedly tried to address the BTTV emote performance on the FFZ repository; still it paints a good view of alternatives and where issues (may) lie.

In hopes of increasing availability and with a look towards the future, I was thinking this would be a good point for discussion. And who know, maybe there are more improvements to be made or found regardless of format used.

**Comment by AnatoleAM** [comment]
We do not use GIF. We use [WEBP](https://developers.google.com/speed/webp)

**Reply by Farcrada** [reply]
Consider my post void.
